### PR TITLE
Simple Addition Program

### PR DESCRIPTION
**Pull request by Grace-Aka**

### CODE REVIEW

**Review by Phildeoner — CHANGES_REQUESTED** [review]
Hello @Grace-Aka 👋, You did a great job.
Just go through the comments below, effect changes and you are good to go👍

File: index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <h1>Simple Addition Program</h1>
+        <div class="info">
+            <p class="text-info">Input an operator by selecting any of the following alphabets below to perform an addition operation.
+                <br>
+                <br>
+                a , b , c , d
```
Comment (Phildeoner) [inline index.html:17]: Since you are creating a calculator for only addition i think just one option will do.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+let a = document.querySelector('.firstInput')
+let b = document.querySelector('.secondInput')
+let result = document.querySelector('.result')
+let btn = document.querySelector('.btn')
+
+function operation() {
+    let newOperator = (selectOperator.value).toLowerCase()
+    let newA = parseInt(a.value)
+    let newB = parseInt(b.value)
+    if (newOperator == 'a') {
```
Comment (Phildeoner) [inline script.js:11]: You need just one operation the repetition is not necessary

**Review by Phildeoner — CHANGES_REQUESTED** [review]
Effect the changes

File: script.js (hunk 2 above)
Comment (Grace-Aka) [reply script.js:11]: okay.

File: index.html (hunk 1 above)
Comment (Grace-Aka) [reply index.html:17]: okay

**Pull request merged**